internal/balancergroup/balancergroup_test.go

```diff
@@ -639,3 +754,84 @@ func (s) TestBalancerGracefulSwitch(t *testing.T) {
 		}
 	}
 }
+
+func (s) TestBalancerExitIdle_All(t *testing.T) {
+	balancerOne := "stub-balancer-test-balancer-group-exit-idle-one"
+	balancerTwo := "stub-balancer-test-balancer-group-exit-idle-two"
+	balancerThree := "stub-balancer-test-balancer-group-exit-idle-three"
```

Comment:
Does three balancers add any more value than two?

And why not?
	balancer1 := t.Name() + "-1"
	balancer2 := t.Name() + "-2"